packages/trace-viewer/vite.config.ts

```diff
@@ -26,6 +26,9 @@ export default defineConfig({
     react(),
     bundle()
   ],
+  define: {
+    'process.env': {},
```

Comment:
Without this, Vite dev server complains about 
https://github.com/Skn0tt/playwright/blob/c0be2d9d1d681221756beae7ac60001ee5be6b42/packages/playwright/src/isomorphic/teleReceiver.ts#L590. I think it's enabled by default in Vite build mode, so we only face it now.